api/v1alpha3/conditions_consts.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+
+const (
+	MachineConditionCount      = 2
+	ControlPlaneConditionCount = 3
+	ClusterConditionCount      = 2
+)
+
+// AzureCluster Conditions and Reasons
+const (
+	NetworkInfrastructureReadyCondition = "NetworkInfrastructureReady"
+	APIServerProvisioningReason         = "APIServerProvisioning"
+	APIServerProvisioningFailedReason   = "APIServerProvisioningFailed"
+	// WaitingForClusterInfrastructureReason used when machine is waiting for cluster infrastructure to be ready before proceeding.
+	WaitingForClusterInfrastructureReason = "WaitingForClusterInfrastructure"
+	// WaitingForBootstrapDataReason used when machine is waiting for bootstrap data to be ready before proceeding.
+	WaitingForBootstrapDataReason = "WaitingForBootstrapData"
```

Comment:
Am I wrong or `WaitingForClusterInfrastructureReason` and `WaitingForBootstrapDataReason` applies to AzureMachine conditions? Should we move this four lines the next conditions block?